alws/crud/errata.py

```diff
@@ -628,6 +768,192 @@ async def get_matching_albs_packages(
     return items_to_insert, package_type
 
 
+async def process_new_errata_references(
+    db: AsyncSession, errata: BaseErrataRecord,
+    db_errata: models.NewErrataRecord, platform: models.Platform
+):
+    references = []
+    self_ref_exists = False
+    for ref in errata.references:
+        db_cve = None
+        if ref.cve:
+            db_cve = await db.execute(
+                select(models.ErrataCVE).where(
+                    models.ErrataCVE.id == ref.cve.id
+                )
+            )
+            db_cve = db_cve.scalars().first()
```

Comment:
we can get these CVE's by one request with something like this:
```python
db_cves = {}
cve_ids = [ref.cve.id for ref in errata.references if ref.cve]
if cve_ids:
  db_cves = {cve.id: cve for cve in await session.execute(...).scalars().all()}
for ref in errata.references:
     ...
    db_cve = db_cves.get(ref)
```

Reply:
done, please check